tmlib/workflow/illuminati/api.py

```diff
@@ -148,16 +148,17 @@ def create_run_batches(self, args):
                             if layer.channel.bit_depth == 8:
                                 if clip_max < 255:
                                     clip_max = 255
+                                    
+                            clip_min = stats.get_closest_percentile(0.001)
                             else:
-                                if clip_max < 700:
-                                    clip_max = 700
+                                if clip_max < clip_min + 255:
+                                    clip_max = clip_min + 255
                             logger.info('clip value: %d', clip_max)
-                            clip_min = stats.get_closest_percentile(0.001)
                         else:
                             logger.info('use provided clip value')
                             clip_max = args.clip_value
                             logger.info('clip value: %d', clip_max)
-                            clip_min = 0
+                            clip_min = stats.get_closest_percentile(0.001)
```

Comment:
I read this as kind of "get closest as possible to zero while still non-zero".  Wouldn't (something like) `clip_min = stats.min()` be better?

Reply:
The 0.001 percentile is a save against dead pixels. There may be very rare pixel with value 0, but we wouldn't care about those normally. 0.001 percentile is typically even too conservative, but it's save.